fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -324,6 +326,9 @@
     @VariableMgr.VarAttr(name = LOAD_MEM_LIMIT)
     public long loadMemLimit = 0L;
 
+    @VariableMgr.VarAttr(name = INSERT_MAX_FILTER_RATIO)
+    public double insertMaxFilterRatio = 0L;
```

Comment:
The default should be 1.0, to be same as origin.
and please update the document:
```
docs/zh-CN/administrator-guide/variables.md
docs/en/administrator-guide/variables.md
```

Reply:
Done